kent_a07b.js

```diff
@@ -0,0 +1,101 @@
+/*
+    Tucker Kent
+    kent_a07b.js
+    19WI_INFO_2124_WW Online Javascript I
+    Thoendel
+    29 January 2020
+*/
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+
+//Step 1
+buildFoodsList (badFoods, "bread", "beer", "liver and onions", "mcdonalds", "lamb"); //added 3 arguments  -- invoking function buildFoodsList       
+
+standardInput.on('data', function (text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    //Step 4 Begin
+if (text === "q" || text === "Q"){ //logical OR operator to accept either case of q/Q on if statement
+    clearScreen(); //invoking function to clear the terminal screen
+    console.log("Bye"); //logging specified bye to the terminal 
+    process.exit(); // invoking function to stop input listening process
+} else if (text === "v" || text === "V"){ //else if statement to accept either case of v/V
+    clearScreen(); //invoking function to clear the terminal screen
+    console.log(displayList(groceryItems)); //invoking function to display grocery list -- logging function with its parameter
+    console.log(getPrompt(groceryItems)); // invoking function to display the food entry prompt -- logging this function with its parameter
+} else { // else statement for any entry other than q or v
+    clearScreen(); //invoking function to clear the terminal screen
+    if(checkItem(text, badFoods) === false){ //nested if statement based on checkItem function return value
+        let item = formatItem(text); // declaring variable item and setting it to return value of function - formatItem
+        groceryItems.push(item); // adding formatted item to groceryItems array using push() method
+        console.log(`${item} added to grocery list.\n\nList contains ${groceryItems.length} ${groceryItems.length === 1? "item" : "items" }\n`); //using template literal to log confirmation with array length
+        console.log(getPrompt(groceryItems)); // logging prompt to terminal -- receiving prompt from invoking function getPrompt()
```

Comment:
love the way you broke up these statements so that you can use that getPrompt()!